examples/grpc-client-bearer-token-auth/grpc_client_bearer_token_auth.out

```diff
@@ -0,0 +1,14 @@
+# Create a Ballerina package.
+# Copy the generated `grpc_secured_pb.bal` stub file to the package.
+# For example, if you create a package named `client`, copy the stub file to the `client` package.
+
+# Create a new `grpc-client-bearer-token-auth.bal` Ballerina file inside the `client` package and add the client implementation.
```

Comment:
Shouldn’t the bal file name should use underscore?

Reply:
Ah yes. I missed that.

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/2083/commits/6ae2cd68bb4b6167e4c9322a5cbf9a0bbcd77ade